sshcode.go

```diff
@@ -548,6 +561,9 @@ chmod +x %v`,
 func ensureDir(path string) error {
 	_, err := os.Stat(path)
 	if os.IsNotExist(err) {
+		if runtime.GOOS == "windows" && strings.HasPrefix(path, "/c/") {
+			path = path[3:]
+		}
```

Comment:
Could you explain why this is needed with a comment?

Reply:
Ah! I dont know how to word this simply but here is what it does

there is a issue on msys2 based environments
ensuredir is fed `/c//<path to ensure>` when running on msys, because of how GO handles said paths, this would end up making `C:\c\<path to ensure>` instead of `C:\<path to ensure>`
What this does, is cut off the `/c/` from the beginning so it is fed `/<path to ensure>` which then gets properly resolved to `C:\<path to ensure>`

It would be helpful if you could find a way to word that simpler then i did. meanwhile i shall flag it as a TODO in my code

Reply:
The has prefix check should be changed from `/c/` to `/c//` since you only want to strip the first 3 chars if there's a slash afterwards.

a good comment would be `Remove /c/ prefix from /c// on windows. This is sometimes an issue in msys2 environments`.